adminplus/sites.py

```diff
@@ -44,7 +44,10 @@ def decorator(fn):
     def get_urls(self):
         """Add our custom views to the admin urlconf."""
         urls = super(AdminPlusMixin, self).get_urls()
-        from django.conf.urls import url
+        try:  # Django 2+
+            from django.urls import re_path as url
+        except:
```

Comment:
Catch the ImportError, it's more specific!

Reply:
You're right, will fix!